__tests__/app.test.js

```diff
@@ -39,4 +39,35 @@ describe("GET /api/categories", () => {
   });
 });
 
-
+describe.only("GET /api/reviews", () => {
+  test("200: retrieves list of reviews from database", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body: { reviews } }) => {
+        expect(reviews).toHaveLength(13);
```

Comment:
you only check the length of the reviews here, I might want to check the structure of them, to ensure they have all the right keys :)